Doc/library/sqlite3.rst

```diff
@@ -424,6 +424,40 @@ Module functions
    Note: *typename* and the name of the type in your query are matched
    case-insensitively.
 
+.. function:: Date(year, month, day)
+
+   This function constructs an object holding a date value.
+
+.. function:: Time(hour, minute, second)
+
+   This function constructs an object holding a time value.
+
+.. function:: Timestamp(year, month, day, hour, minute, second)
+
+   This function constructs an object holding a time stamp value.
+
+.. function:: DateFromTicks(ticks)
+
+   This function constructs an object holding a date value from the given ticks
+   value (number of seconds since the epoch; see the documentation of the
```

Comment:
I assume it's the UNIX epoch and not Gregorian epoch? Can it be a fractional number of seconds or not?

Reply:
> I assume it's the UNIX epoch and not Gregorian epoch?

Yeah the UNIX epoch.

> Can it be a fractional number of seconds or not?

This was taken from PEP-249. Should we get clarification there?

Reply:
I think it was not considered at that time. In general, we don't change PEPs (unless there is a real issue with the PEP I think, but I am not aware of a precedent). Best is to ask @erlend-aasland: should we leave it imprecise, or is there some assumptions that I'm not aware of?